.gitignore

```diff
@@ -10,6 +10,8 @@ build/
 artifacts/
 src/**/obj/
 src/**/bin/
+buildscripts/**/obj/
+buildscripts/**/bin/
```

Comment:
If you open the solution up in VS2017, you will see that I have included all the build scripts/config inside the solution. Thought this would be handy for doing build work, rather than diving out to the file system but it needs gitignores for bin/obj output. 

![solution-layout](https://cloud.githubusercontent.com/assets/607402/24992900/a90060ae-201b-11e7-8a9d-522c7fc1b8a6.png)